types/repo/AnonymousMessageRepo/AnonymousChannelsData.ts

```diff
@@ -70,7 +74,7 @@ export interface Message {
   channelId?: string;
 }
 
-export interface AnonymousChannelData {
+export interface ChannelData {
   id: string;
   type: string;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `AnonymousChannelData` interface has been renamed to `ChannelData`. Make sure to update all references to this interface in your codebase to avoid any type errors.

```diff
- export interface AnonymousChannelData {
+ export interface ChannelData {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->